src/backend/launcher.ts

```diff
@@ -674,9 +674,16 @@ async function prepareWineLaunch(
       writeFileSync(appsNamesPath, JSON.stringify([appName]), 'utf-8')
       hasUpdated = true
     } else {
-      const installedGames: string[] = JSON.parse(
-        readFileSync(appsNamesPath, 'utf-8')
-      )
+      let installedGames: string[] = []
+
+      try {
+        installedGames = JSON.parse(
+          readFileSync(appsNamesPath, 'utf-8')
+        ) as string[]
```

Comment:
In cases like this, where it's easy to add a runtime type-check, I'd say we should do so.
```suggestion
        installedGames = z.string().array().parse(JSON.parse(
          readFileSync(appsNamesPath, 'utf-8')
        ))
```
(missing `import z from 'zod'` and probably won't lint, but you get the idea)